client/js/components/report/DpReportListing.vue

```diff
@@ -16,7 +16,9 @@
     <div
       class="float-right u-mt-0_25"
       v-if="hasPermission('feature_export_protocol')">
-      <a :href="Routing.generate('dplan_export_report', { procedureId })">
+      <a
+        data-cy="exportTriggerPdf"
```

Comment:
Shouldn't it be something like
```suggestion
        data-cy="reportListing:exportTriggerPdf"
```
?

Reply:
not sure, hence the whole page is reportingList. But it doesn't hurt to add it.